src/Microsoft.XmlSerializer.Generator/src/Sgen.cs

```diff
@@ -118,6 +122,10 @@ private int Run(string[] args)
                     {
                         warnings = true;
                     }
+                    else if (ArgumentMatch(arg, "reference"))
+                    {
+                        references = value;
+                    }
```

Comment:
What happens if multiple /reference parameters are passed on the commandline. You might want to do `reference = string.IsNullOrEmpty(reference) ? value : reference + ";" + value;`

Reply:
We only support one reference, which should be mentioned in our doc. And that's also what full framework did and I feel we'd better follow the same rule.